controllers/suite_v2_test.go

```diff
@@ -0,0 +1,152 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build integration_v2 && !integration
+// +build integration_v2,!integration
+
+// Note: This is very similar to "suite_test.go". The only differences are that
```

Comment:
For reviewers, this is a big file, but it's mostly identical to the existing one (`suite_test.go`) and it will replace it once the migration is done.